llvm/include/llvm/Support/Casting.h

```diff
@@ -841,6 +877,22 @@ template <typename... Types>
 inline constexpr detail::IsaAndPresentCheckPredicate<Types...>
     IsaAndPresentPred{};
 
+/// Function objects corresponding to the Cast types defined above.
+template <typename... Types>
+inline constexpr detail::StaticCastFunc<Types...> StaticCastTo{};
```

Comment:
These casts operators are not variadic

```suggestion
template <typename T>
inline constexpr detail::StaticCastFunc<T> StaticCastTo{};
```